src/patternfly/components/Breadcrumb/breadcrumb.scss

```
@@ -84,3 +84,9 @@
     font-weight: var(--pf-global--FontWeight--semi-bold);
   }
 }
+
+// Hide divider of first child
```

Adding a note here for why it's necessary since it isn't intuitive. Also a "breaking change" note so we can revisit this later and try and remove the unnecessary HTML.

```suggestion
// Hide divider of first child - required for react since dividers are included in each item
// Consider revisiting this in a breaking change release
```